### DISCUSSION TITLE
Is it possible to use ISO-8859 charset?

### DISCUSSION THREAD

**Discussion by grudnitzki** [TarekRaafat/autoComplete.js#324] [Q&A]
Both my JSON results file and my html file use ISO-8859 charset.  Searches with accented characters do not return results. How can I fix it?

**Comment by TarekRaafat** [comment]
Hello @grudnitzki,

Have you tried the [`diacritics`](https://tarekraafat.github.io/autoComplete.js/#/configuration?id=diacritics-optional) config?

**Comment by grudnitzki** [comment]
Yes, I tried the [diacritics](https://tarekraafat.github.io/autoComplete.js/#/configuration?id=diacritics-optional) config, but it didn't help. I also tried the query option, but it seems it is never called. I put a "hello" console.log inside of the query function and it does not show in the console.

**Reply by TarekRaafat** [reply]
Could you please share a sample of the used data and query?

**Comment by grudnitzki** [comment]
Please see a sample [here](https://json.lojavirtualfc.com.br/lojas/00043907/htm/autocomplete.htm).

JSON data is [here](https://json.lojavirtualfc.com.br/lojas/00043907/htm/autocomplete.js).

If you search for **abacaxi calça**, nothing is found. But if you manually encode de ç in the search field, it works:

<img width="319" alt="image" src="https://user-images.githubusercontent.com/19479100/158079583-a34fdb6a-655e-47aa-881b-5b5b5d97e328.png">

**Reply by trasherdk** [reply]
Do you have one of these in your `head` ? Could that be related?
`<meta charset="utf-8">`

**Reply by trasherdk** [reply]
Response from server :smiley: 
```
Object {
  food: "Abacaxi Cal&#231;a",
  cities: "Cerna",
  animals: "Pintail, bahama"
}
```

**Comment by grudnitzki** [comment]
We use ISO-8859 charset, not utf-8;

**Comment by trasherdk** [comment]
You are requesting `abacaxi calça` and the server returns `Abacaxi Cal&#231;a` So, no match.
